translate/storage/mozilla_lang.py

```diff
@@ -108,7 +108,9 @@ def parse(self, lines):
                 self.is_active = True
                 continue
 
-            if line.startswith("## ") and not line.startswith('## TAG'):
+            if (line.startswith("## ")
+                and not line.startswith('## TAG')
+                and not line.startswith('## MAX_LENGTH')):
```

Comment:
can we break up the multiliine conditions as its easy to miss that they are conditions or where the clause is etc